packages/scheduler/src/components/FullCalendar/FullCalendar.js

```diff
@@ -125,29 +110,134 @@ class FullCalendar extends React.Component {
   }
 
   componentDidMount() {
-    this.calendar = $(this.calendarContainer)
-    this.calendar.fullCalendar(this.getFullCalendarOptions(this.props))
+    this.calendarElement = $(this.calendarElementRef)
+    this.calendarElement.fullCalendar(this.getFullCalendarOptions(this.props))
+    this.calendar = this.calendarElement.fullCalendar('getCalendar')
+    this.handleRangeChange()
+    this.forceUpdate()
   }
 
   componentWillReceiveProps(newProps) {
-    this.calendar.fullCalendar('option', this.getFullCalendarOptions(newProps))
+    this.calendarElement.fullCalendar('option', this.getFullCalendarOptions(newProps))
 
     if (!_isEqual(newProps.events, this.props.events)) {
-      this.calendar.fullCalendar('removeEventSources')
-      this.calendar.fullCalendar('addEventSource', newProps.events)
+      this.calendarElement.fullCalendar('removeEventSources')
+      this.calendarElement.fullCalendar('addEventSource', newProps.events)
     }
 
     if (!_isEqual(newProps.resources, this.props.resources)) {
-      this.calendar.fullCalendar('refetchResources')
+      this.calendarElement.fullCalendar('refetchResources')
+    }
+  }
+
+  changeView = (view = 'timelineDay') => {
+    this.calendarElement.fullCalendar('changeView', view)
+    this.handleRangeChange()
+  }
+
+  getButtonInkProps = viewType => {
+    const currentViewType = this.calendarElement ? this.calendarElement.fullCalendar('getView').type : ''
+
+    if (currentViewType === viewType) {
+      return {ink: 'primary'}
     }
 
-    this.calendar.fullCalendar('option', 'customButtons', {
-      refresh: this.getRefreshButton(newProps.isLoading)
-    })
+    return {}
   }
 
-  render = () =>
-    <div className={this.state.wrapperId}><div ref={ref => { this.calendarContainer = ref }}></div></div>
+  msg = (id, values = {}) => (this.props.intl.formatMessage({id}, values))
+
+  render = () => {
+    // TODO: Remove ternary operator as soon as Child is no more a required prop
```

Comment:
Child is not required anymore right? Remove comment and adjust as described plase